spk/rutorrent/src/installer.sh

```diff
@@ -147,6 +183,22 @@ postinst ()
     chown -R ${USER}:${APACHE_USER} ${INSTALL_DIR}/tmp
     chown -R ${APACHE_USER}:${APACHE_USER} ${WEB_DIR}/${PACKAGE}
 
+    # Fix rights using ACL (so that test.sh remains executable)
+    if [ "${BUILDNUMBER}" -ge "7321" ]; then
+        set_acl_full_permissions "${WEB_DIR}/${PACKAGE}" "user:${USER}"
```

Comment:
Ok, so basically the issue here was that `rutorrent` (running with user `http`) relies on `rtorrent` (running with user `sc-rutorrent`) to execute a file under `php/test.sh` to detect / valid external programs.
So here we basically need to ensure that (following rutorrent official documentation) web installation directory is accessible to `sc-rutorrent` with full rights. 
And, unfortunately, it seems that for Synology it's either you're going all in on ACLs or you're using standard Linux rights. So I needed to give full acl permissions to sc-rutorrent and web user / groups.